client/components/Main.jsx

```diff
@@ -83,6 +85,9 @@ export default class Main extends React.Component {
     super(props, context);
 
     this.socket = props.socket;
+    this.match = props.match;
+    this.location = props.location;
+    this.history = props.history;
```

Comment:
Why are these values set?

Reply:
you are right, this is not needed, will remove these